src/commands/color.ts

```diff
@@ -0,0 +1,31 @@
+import { Client, Message } from 'discord.js';
+
+exports.run = (client: Client, msg: Message, args: string[]) => {
+  const roleName = 'USER-' + msg.author.id;
+  const myRole = msg.guild.roles.find((role) => role.name === roleName);
+  let chosenColor = args[0].toUpperCase();
+
+  if (!/(^#?[0-9A-F]{6}$)|(^#?[0-9A-F]{3}$)/i.test(chosenColor)) { // Ensures first argument is a valid hex color
```

Comment:
For readability, can you set this regex to a const on the line above the if statement? Prob the same with the other regex patterns in here. The commented code helps a lot tho